langkit/count_regexes.py

```diff
@@ -0,0 +1,76 @@
+from logging import getLogger
+
+from langkit.pattern_loader import PatternLoader
+from whylogs.experimental.core.udf_schema import register_dataset_udf
+from . import LangKitConfig
+from whylogs.core.stubs import pd
+from typing import Dict, List, Optional, Set, Union
+
+diagnostic_logger = getLogger(__name__)
+
+
+pattern_loader = PatternLoader()
+
+
+def count_patterns(group, text: str) -> int:
+    count = 0
+    for expression in group["expressions"]:
+        if expression.search(text):
+            count += 1
+
+    return count
+
+
+def wrapper(pattern_group, column):
+    def wrappee(text: Union[pd.DataFrame, Dict[str, List]]) -> Union[pd.Series, List]:
+        return [count_patterns(pattern_group, input) for input in text[column]]
+
+    return wrappee
+
+
+_registered: Set[str] = set()
+
+
+def _unregister():
+    # WARNING: UNSUPPORTED HEINOUS EVIL
```

Comment:
Suggest rephrasing, and add a TODO for the follow up work required to make this something that can be done cleanly?